tests/e2e/data/services/test_bq_datasets.py

```diff
@@ -1,241 +0,0 @@
-import os
```

Comment:
These are good e2e tests and is a pity to lose them. Is there any chance of either modify them to run through the EnrichmentService or move them to `carto-python`?

Reply:
Let's talk offline about them

Reply:
Added